mappings/net/minecraft/command/arguments/BlockStateArgument.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS db net/minecraft/command/arguments/BlockStateArgument
```

Comment:
Should the name reflect that it also contains BE data?

Reply:
Good point, I overlooked that. Ideally, yes, this could be a more generic `BlockArgument`, and the type would be renamed to `BlockArgumentType`.

Reply:
This also means that `BlockStateArgumentParser` should be renamed to `BlockArgumentParser`